cdp_scrapers/instances/portland.py

```diff
@@ -37,7 +37,9 @@
 
 ###############################################################################
 
-SCRAPER_STATIC_DATA = parse_static_file(Path(__file__).parent / "portland-static.json")
+SCRAPER_STATIC_DATA = parse_static_file(
+    Path(__file__).parent / "portland-static.json", "America/Los_Angeles"
```

Comment:
I know this PR isn't that place to discuss this but I am going to note here that it seems like some of the static data JSON files are out of date. I think Portland specifically but once the backfill is done I will try to compile a list to update -- I will help update them too hahah